components/logins/src/db.rs

```diff
@@ -30,22 +30,22 @@ use url::{Host, Url};
 
 #[derive(Serialize, Deserialize, PartialEq, Debug, Clone, Default)]
 pub struct MigrationPhaseMetrics {
-    num_processed: u64,
-    num_succeeded: u64,
-    num_failed: u64,
-    total_duration: u128,
-    errors: Vec<String>,
+    pub num_processed: u64,
+    pub num_succeeded: u64,
+    pub num_failed: u64,
+    pub total_duration: u64,
```

Comment:
Unfortunately, Uniffi doesn't have i/u greater than 64 but it looks like it was chosen as u128 since that was what Rust defaults to when using the is_milis() so it seems like we might be okay

Reply:
I just git blamed that line and I actually chose `u128` when I created the `MigrationPhaseMetrics` and if memory serves it was for the exact reason you specified. Also, (this)[https://github.com/mozilla/application-services/blob/72b827c3e0f883163762857fd766df1aeb060725/components/support/ffi/src/into_ffi.rs#L48] probably explains why Uniffi doesn't support `u128`.

Reply:
I also bet noone looks at these metrics these days :)